tasks/configure-Debian.yml

```diff
@@ -38,6 +38,17 @@
   notify: restart apache
   when: apache_create_vhosts
 
+- name: Configure Apache (Add listen port for every vhost).
+  blockinfile:
+    dest: "{{ apache_server_root }}/conf/{{ apache_daemon }}.conf"
+    insertafter: "^Listen {{ apache_listen_port }}"
+    block: "{% set return = joiner(\"\\n\") %}{% for item in apache_vhosts %}{% if item.listenport is defined and apache_listen_port != item.listenport %}{{ return() }}Listen {{ item.listenport }}{% endif %}{% endfor %}"
```

Comment:
I think this could be a lot simpler if there was just a new variable like `apache_listen_ports_additional`, which would allow an array of ports to be listed.

You could set listen ports per vhost, but then you would also need to add each of the additional ports to the new variable.

The problem is, I don't think I'd be willing to merge this change as-is, because it's fairly complex logic to support addition of extra ports (trying to read through this `block` line is quite a task!).